types/normalize-port/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for normalize-port 1.0
+// Project: https://github.com/aoberoi/normalize-port.js
+// Definitions by: Alex Alonso <https://github.com/finaldes>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare module "normalize-port" {
```

Comment:
That can be determined from the file name -- you don't need the `declare module` too.